### PR TITLE
Describe how to test potential contributions to the DITA-OT

### PR DESCRIPTION
**Pull request by chrispy-snps**
## Description
Updates the Developers section of the main `README.markdown` file to describe how to test potential contributions,

## Motivation and Context
As a novice contributor, I was unsure of how to test potential contributions to the DITA-OT. I wanted to capture this information.

Questions for those more knowledgeable:

* Are my comment descriptions correct?
* Is the test ordering correct, or should they be run in a different order?
* Are any tests missing?

## How Has This Been Tested?
No tests needed.

## Type of Changes
Small README change only; no code changes.

## Checklist
I tried to follow the existing style of the `README.markdown` file.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -98,6 +98,12 @@ See the [documentation][docs] for arguments and [options].
 
     For more information on the `-Xmx` option, see the [Java SE Documentation][javadoc].
 
+3.  For potential contributions to the project, test the changes:
+
+        ./gradlew test             # run module unit tests
+        ./gradlew integrationtest  # run module integration tests
+        ./gradlew e2etest          # run end-to-end tests
+
```
Comment (jelovirt) [inline README.markdown:106]: IMO these should not be under "Distribution builds" section. A more suitable place would be as the 5th step for "For developers" section.

**Review by jelovirt — CHANGES_REQUESTED** [review]

File: README.markdown (hunk 1 above)
Comment (chrispy-snps) [reply README.markdown]: Silly error fixed.

**Comment by chrispy-snps** [comment]
@jelovirt - does `./gradlew dist` need to be run before the tests are run? (I now recall that this is why I originally placed them in the "Distribution builds" section.)

**Comment by jelovirt** [comment]
Implemented in #4243

**Comment by chrispy-snps** [comment]
Thanks @jelovirt!

**Pull request closed without merging**